scania-digital-design-system/sdds-website
# Improvement/task 1621

**Describe pull-request**  
- Installed prettier for code-formatting
- Installed eslint (and configured rules) for linting
- Installed husky for pre-commit hooks
- prettier and eslint automatically runs on commit to avoid errors in committed code.

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #1621
(also fixes 1622 and 1624)

**How to test**  
Check out the branch, do a commit. If that works fine, make an intended error (or mess up formatting), and retry. should fail :)
